src/app/app.module.ts

```diff
@@ -1,18 +1,20 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
-import { AppComponent } from './app.component';
-import { UserHelpComponent } from './user-help/user-help.component';
+import { AppComponent } from './components/root/app.component';
+import { FaqAssistantComponent } from './components/faq-assistant/faq-assistant.component';
+import { TitleService } from './services/title/title.service';
+import {ItemManager} from './item-manager/item-manager';
```

Comment:
Different styles among import `{ItemManager}` vs `{ TitleService }`. Please, choose the one and follow them. 
Also you can setup linter and it will highlight these moments.

Reply:
was applied